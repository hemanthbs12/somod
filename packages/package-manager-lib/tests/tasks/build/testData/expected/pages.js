export { default as Page1 } from "./pages/Dashboard";
export { default as Page2 } from "./pages/DialogStepper";
export { default as Page3 } from "./pages/EntryLayout";
export { default as Page4 } from "./pages/Page";
export { default as Page5 } from "./pages/yupExtension/View";
export { default as Page6, DateTimeSchema as Page6DateTimeSchema } from "./pages/yupExtension/schemas/DateTime";
export { default as Page7, FileSchema as Page7FileSchema } from "./pages/yupExtension/schemas/File";
export { stringSchema as Page8stringSchema, numberSchema as Page8numberSchema, booleanSchema as Page8booleanSchema, datetimeSchema as Page8datetimeSchema, fileSchema as Page8fileSchema, appendRequiredOnYupSchema as Page8appendRequiredOnYupSchema, describeFields as Page8describeFields, getFieldLabels as Page8getFieldLabels, getFieldTypes as Page8getFieldTypes, reverseKeyOrder as Page8reverseKeyOrder } from "./pages/yupExtension/utils";