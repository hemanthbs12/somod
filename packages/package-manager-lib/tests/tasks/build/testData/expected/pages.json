{
  "Dashboard": {
    "prefix": "Page1",
    "exports": {
      "default": true,
      "named": [
        "DashboardProps"
      ]
    }
  },
  "DialogStepper": {
    "prefix": "Page2",
    "exports": {
      "default": true,
      "named": [
        "DialogStepperProps"
      ]
    }
  },
  "EntryLayout": {
    "prefix": "Page3",
    "exports": {
      "default": true,
      "named": [
        "EntryLayoutProps"
      ]
    }
  },
  "Page": {
    "prefix": "Page4",
    "exports": {
      "default": true,
      "named": [
        "PageProps"
      ]
    }
  },
  "yupExtension/View": {
    "prefix": "Page5",
    "exports": {
      "default": true,
      "named": []
    }
  },
  "yupExtension/schemas/DateTime": {
    "prefix": "Page6",
    "exports": {
      "default": true,
      "named": [
        "Variant",
        "DateTimeSchema"
      ]
    }
  },
  "yupExtension/schemas/File": {
    "prefix": "Page7",
    "exports": {
      "default": true,
      "named": [
        "FileType",
        "FileSchema"
      ]
    }
  },
  "yupExtension/utils": {
    "prefix": "Page8",
    "exports": {
      "default": false,
      "named": [
        "SupportedType",
        "DatetimeVariant",
        "stringSchema",
        "numberSchema",
        "booleanSchema",
        "datetimeSchema",
        "fileSchema",
        "appendRequiredOnYupSchema",
        "describeFields",
        "getFieldLabels",
        "getFieldTypes",
        "reverseKeyOrder"
      ]
    }
  }
}